tests/providers/apache/cassandra/hooks/test_cassandra_hook.py

```diff
@@ -30,15 +31,7 @@
 from airflow.utils import db
 
 
-def cassandra_is_not_up():
-    try:
-        Cluster(["cassandra"])
-        return False
-    except UnresolvableContactPoints:
-        return True
-
-
-@unittest.skipIf(cassandra_is_not_up(), "Cassandra is not up.")
+@pytest.mark.integration("cassandra")
 class TestCassandraHook(unittest.TestCase):
```

Comment:
If the reason behind Cassandra flakiness was lack of resources then this is should work.

Reply:
I truly hope so! It looks like so far.